extra/3-magic-8-ball.js

```diff
@@ -59,9 +88,25 @@ function shakeBall() {
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
 function checkAnswer(answer) {
-  //Write your code in here
+  let message = "";
```

Comment:
There is an easier way of doing this using indexOf.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf

Reply:
Thanks Karen this link makes my code more simple and understand. I've learned more more from here. Thanks for sharing. I've replace my code with using .indexOf and its pass the test! hurray!